web/client/epics/map.js

```diff
@@ -61,7 +66,45 @@ const handleCreationLayerError = (action$, store) =>
         ]) : Rx.Observable.empty();
     });
 
+const startEditing = (action$, store) =>
+    action$.ofType(FEATURE_EDITING)
+    .filter(() => store.getState().featuregrid && store.getState().featuregrid.select && store.getState().featuregrid.select.length > 0)
+    .switchMap(() => {
+        const isLeaflet = store.getState().maptype.mapType === "leaflet";
+        const defaultFeatureProj = "EPSG:4326";
+        let newFeatures;
+        let feature = head(store.getState().featuregrid.select); // just to be sure there is one feature selected for the editing
+        let drawOptions = {
+            featureProjection: defaultFeatureProj,
+            stopAfterDrawing: true,
```

Comment:
these should be deduced from the action payload (e.g. `create` or `edit`  flags)